pkg/virt-config/config-map.go

```diff
@@ -196,6 +197,7 @@ func defaultClusterConfig() *v1.KubeVirtConfiguration {
 		SELinuxLauncherType:         DefaultSELinuxLauncherType,
 		SupportedGuestAgentVersions: supportedQEMUGuestAgentVersions,
 		OVMFPath:                    DefaultOVMFPath,
+		MemBalloonStatsPeriod:       DefaultMemBallonStatsPeriod,
```

Comment:
Do we want, DefaultMemBallo**o**nStatsPeriod   here?

Reply:
Yup, nice catch :)

Reply:
@kbidarkar @andreabolognani can you elaborate?

Reply:
@xpivarc There's a typo in `DefaultMemBallonStatsPeriod`, it should be `DefaultMemBalloonStatsPeriod` (note the two "o"s).